D99-D100 AADVT instrumentation (ATMDYN.f part: ffdc_aadvt_in/_out around the live CALL AADVT in DYNAM, ATMDYN.f:337). Helpers are in ATM_DRV_dynC.f.patch (units 1100-1102). Generated by diff -u against the PRISTINE ATMDYN.f (2026-10-05); one local hunk, applies to pristine or after the other ATMDYN_*.patch files (disjoint region).

--- ATMDYN.f.orig
+++ ATMDYN.f
@@ -334,7 +334,9 @@
       TZT(:,:,:) = TZ(:,:,:)
       Do L=1,LM
          MMA(:,:,L) = MEVEN(L,:,:)*AXYP(:,:)  ;  EndDo
+      call ffdc_aadvt_in(DTLF,MMA,T,TMOM)
       Call AADVT (DTLF, MMA,T,TMOM, .False., FPEU,FPEV)
+      call ffdc_aadvt_out(MMA,T,TMOM,FPEU,FPEV)
 #ifdef V2_ATMDYN_TIMESTEPPING
       Call VDIFF  (DTLF, UT,VT,             U,V,MA, T)
 #endif
